code/addons/onboarding/preset.js

```diff
@@ -0,0 +1,8 @@
+function managerEntries(entry = []) {
```

Comment:
I know this PR is just porting over the existing code, but any idea why this is needed?

Reply:
I have no idea.

We'd have to ask the authors @yannbf or @valentinpalkovic ?

Reply:
@shilman Could you please be a bit more precise about what exactly you are referring to? Are you saying that the managerEntries should just be exported in the `src/preset.ts` file like described here: https://storybook.js.org/docs/addons/writing-presets#root-level-presets? I don't understand the intention of your question.